selfservice/strategy/password/.schema/registration.schema.json

```diff
@@ -15,6 +15,10 @@
     },
     "method": {
       "type": "string"
+    },
+    "transient_payload": {
+      "type": "object",
+      "additionalProperties": true
```

Comment:
Just FYI, `additionalProperties` defaults to true. But I think it makes sense to have this explicit here.

Reply:
:+1: